R/plot_recurrent_genes.R

```diff
@@ -74,15 +98,23 @@ plot_recurrent_genes <- function(standardFusioncalls,
     palette_2 <- palette[which(names(palette) %in% rec_gene[, groupby])]
   }
 
+  if (base_size != 12) {
+    base_size <- base_size
+  }
+
   rec_genes <- ggplot(rec_gene) +
     geom_col(aes(x = .data$Gene, y = .data$count, fill = !!as.name(groupby)), alpha = 0.75) +
-    ylab("Number of patients") +
+    ylab(paste0("Number of ", countID)) +
     guides(color = FALSE, alpha = FALSE) +
     xlab(NULL) +
-    scale_y_continuous(breaks = seq(0, 150, by = 10)) +
+    scale_y_continuous(breaks = seq(0, max(rec_gene$count), by = 10)) +
```

Comment:
same suggestion as above

```suggestion
    scale_y_continuous(limits = (0, max(rec_gene$count))) +
```